### DISCUSSION TITLE
how to keep column names with underscore in it

### DISCUSSION THREAD

**Discussion by guido-gtt** [linq2db/linq2db#4673] [Q&A] [answered]
```
public partial class TblRelation
{
	[Column(),         PrimaryKey,  Identity] public int       RelationID                                { get; set; } // int
	[Column("SCS_ID"),    Nullable          ] public int?      ScsId                                     { get; set; } // int
```

the database schema that is created has renamed my column SCS_ID to ScsId, I do not want that, how can I avoid this ?
What setting (and in what file) do I have to alter for this ?

**Comment by MaceWindu** [comment, marked as answer]
Check `NormalizeNames` option https://linq2db.github.io/articles/T4.html#configuring-generation-process
